omnisafe/envs/core.py

```diff
@@ -199,6 +200,7 @@ def __init__(self, env: CMDP, device: torch.device = DEVICE_CPU) -> None:
         """Initialize an instance of :class:`Wrapper`."""
         self._env: CMDP = env
         self._device: torch.device = device
```

Comment:
add attributes descriptions for `self._device` and `self.need_evaluation`

Reply:
fixed